firmware/quadruna/BMS/test/test_stateMachine.cpp

```diff
@@ -359,14 +359,29 @@ TEST_F(BmsStateMachineTest, charger_connected_can_msg_init_state)
     fake_io_airs_isNegativeClosed_returns(true);
 
     // Simulate situation with charger present and user indicate to start charging
-    fake_io_charger_isConnected_returns(true);
+    app_canRx_BRUSA_IsConnected_update(true);
     app_canRx_Debug_StartCharging_update(true);
 
     LetTimePass(210U);
 
     ASSERT_EQ(app_prechargeState_get(), app_stateMachine_getCurrentState());
 }
 
+// For now this isn't working, we'll have to rethink how to use missing_hb
+// to stop the BMS going into drive state when charger is not connected
+TEST_F(BmsStateMachineTest, no_charger_connected_missing_hb_init_state)
+{
+    SetInitialState(app_initState_get());
+    fake_io_airs_isNegativeClosed_returns(true);
+    app_heartbeatMonitor_blockFaults(false);
+    app_canRx_VC_Heartbeat_update(false);
+    app_canRx_RSM_Heartbeat_update(false);
+
+    LetTimePass(500U);
+
+    ASSERT_EQ(app_initState_get(), app_stateMachine_getCurrentState());
+}
```

Comment:
Is this still not working or is this outdated?

Reply:
I fixed the test, forrgot about the comment. thanks for the reminder